packages/interpret-cli/src/transfer/bean/to-vo.ts

```diff
@@ -62,21 +64,30 @@ export async function toBeanClass(
     if (typeDef.fields[fieldName].isArray) {
       filedType = typeDef.fields[fieldName].elementType.name;
     }
+    const regex = new RegExp(
+      `^(get|set${fieldName.slice(0, 1).toUpperCase()}${fieldName.slice(1)})$`,
```

Comment:
Please consider below test case:

```js
const fieldName = 'name';
const regex = new RegExp(`^(get|set${fieldName.slice(0, 1).toUpperCase()}${fieldName.slice(1)})$`);
regex.test('getName'); // expect `true` but got `false`
```

and then if we just want to capitalize the first character I think the better way is using `fieldName[0].toUpperCase()` directly, it's safe since `fieldName` can never be a empty string

Reply:
Oh, sorry. It's a typo. Please review it again.

Reply:
LGTM. It's up to @hufeng 😏

Reply:
OK